src/components/ui4/RowUi4.tsx

```diff
@@ -70,7 +70,7 @@ export const RowUi4 = (props: Props) => {
       try {
         await onLongPress()
       } catch (err) {
-        showError(err)
+        showDevError(err)
```

Comment:
Since this is already inside `EdgeTouchableOpacity`, just delete the whole `try / catch` block.